lib/cram.mli

```diff
@@ -20,8 +20,17 @@ type t = {
   command : string list;
   output : Output.t list;
   exit_code : int;
+  (* TODO: remove vpad *)
```

Comment:
Good point. I had it set to 0 temporarily, since there's no more padding between cram tests (the padding is now part of another type), but indeed, now that I'm done I can remove it as it is not coming back.